workspace/lib/lita/handlers/jenkins.rb

```diff
@@ -0,0 +1,48 @@
+require 'jenkins_api_client'
+require 'jenkinshelper/jenkins'
+module Lita
+  module Handlers
+    class Jenkins < Handler
+      config :ip, required: true, type: String
+      config :username, required: true, type: String, default: ''
+      config :password, required: true, type: String, default: ''
+
+      include ::JenkinsHelper::Jenkins
+
+      route(
+         /.*build_jenkins(.*)/,
+         :build_job_now,
+         command: false,
+         help: {t('help.build_jenkins.syntax') => t('help.build_jenkins.desc')}
+      )
+
+      def build_job_now(response)
+        argument = response.matches[0][0].strip
+        job_name, params = argument.split(' ')
+        response.reply(t('build_jenkins.building', job_name: job_name))
+        begin
+          if job_exists?(job_name)
+            if (params.nil? or params.empty?)
+              client.api_post_request("/job/#{path_encode job_name}/build", {})
+            else
+              client.api_post_request("/job/#{path_encode job_name}/buildWithParameters", params)
+            end
+            queue_result = get_item_in_queue(job_name)
+            unless queue_result == false
```

Comment:
Suggest: We can edit it 
<code>if queue_result</code>

Reply:
I want to the expectation path executed first, then the exception